app/services/doc_auth/socure/requests/document_request.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+module DocAuth
+  module Socure
+    module Requests
+      class DocumentRequest < DocAuth::Socure::Request
+        attr_reader :document_type, :redirect_url, :document_capture_session_uuid, :language
+
+        def initialize(
+          document_capture_session_uuid:,
+          redirect_url:,
+          language:,
+          document_type: 'license'
+        )
+          @document_capture_session_uuid = document_capture_session_uuid
+          @redirect_url = redirect_url
+          @document_type = document_type
+          @language = language
+        end
+
+        private
+
+        def lang(language)
+          return language unless language == :zh
+          'zh-cn'
```

Comment:
it is possible we could add other languages ... it think having a condition for the exception is a more flexible path forward than returning the exception by default 🤔 

```suggestion
          return 'zh-cn' if language == :zh
          language
```